packages/contracts/contracts/plugin/PluginSetupProcessor.sol

```diff
@@ -105,16 +106,17 @@ contract PluginSetupProcessor is DaoAuthorizable {
     }
 
     /// @notice The struct containing the parameters for the `prepareUninstallation` function.
-    /// @param pluginSetupRef The PluginSetupRepo address + the current version of the plugin at which time it's getting uninstalled.
-    /// @param setupPayload see IPluginSetup.SetupPayload
+    /// @param pluginSetupRef The reference to the plugin setup to be used for the uninstallation.
+    /// @param setupPayload The payload containing the existing contracts as well as optional data to be consumed by the plugin setup.
```

Comment:
maybe instead of `existing contracts`, better to say `the plugin address and related helpers deployed by the previous preparation which was actually applied`.

Reply:
Good suggestion. Applied in d462d56b3e691ba377589d044fbe94efbf83ac13